inc/ubm_common.h

```diff
@@ -89,6 +89,8 @@
 //BP unique ID
 #define BP_ID_NONE                                                  0xC0
 #define BP_ID_2U_2_5_Anybay_8_Bay                                   0xC1
+#define BP_ID_2U_U3_Anybay_8_Bay                                    0xD1
```

Comment:
Can you please define the macros within parentheses?
#define BP_ID_2U_U3_Anybay_8_Bay (0xD1)